.github/workflows/modmesh_viewer.yml

```diff
@@ -40,6 +40,21 @@ jobs:
             libgl1-mesa-dev
 
         sudo pip3 install numpy pytest flake8
+        
+    - name: dependency (macos)
+      if: ${{ startsWith(matrix.os, 'macos-12') }}
+      run: |
+        # brew install python3 llvm
```

Comment:
Why do we use the comments here?

Reply:
Do you mean merging from, for example, `add-viewer-macos-ci` to `master`?

Btw about the comments, I forgot to remove them before pushing. I will fix it afterwards.

Reply:
Yes that's what I meant, and I saw your new PR #117